components/button/test/floating-buttons.visual-diff.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../typography/typography.js';
+		import '../../demo/demo-snippet.js';
+		import '../button.js';
+		import '../floating-buttons.js';
+	</script>
+	<title>d2l-floating-buttons</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<style>
+	</style>
+</head>
+<body class="d2l-typography">
+		<d2l-demo-snippet id="floating-buttons-short">
```

Comment:
I used demo snippets in this test page because they allowed for multiple floating button situations on one page.